intro/numpy/exercises.Rmd

```diff
@@ -40,7 +40,7 @@ import matplotlib.pyplot as plt
 
 and generate a new array containing its 2nd and 4th rows.
 
-### Divide each column of the array:
+**Divide each column of the array:**
```

Comment:
Why this change?

Reply:
It as creating an end node error and breaking the build?

Reply:
I've removed the junk as well, though I did like the cat addition!

Reply:
Hmm - that's strange - any idea why a heading was breaking the build?